src/modules/commander/Commander.cpp

```diff
@@ -894,6 +894,10 @@ Commander::handle_command(const vehicle_command_s &cmd)
 					} else {
 						arming_res = disarm(arm_disarm_reason_t::COMMAND_INTERNAL);
 					}
+
+					if (arming_res != TRANSITION_DENIED && !_land_detector.landed) {
```

Comment:
I think we should check explicitly for `TRANSITION_CHANGED`. Triggering the parachute on `TRANSITION_NOT_CHANGED` can potentially lead to dangerous situations where the drone receives a disarm signal while disarmed on the ground, with unsuspecting ops personnel nearby. What do you think?

Reply:
Fixed.
It should not matter, as we test for landed as well, but should be good like this.

We will probably need to implement some way of disarming the parachute when landing later. There should probably be some procedure "do not approach the drone without verifying that the parachute is disarmed (LED or QGC)".

Reply:
Good point. I will make sure that proper parachute-handling procedures are part of the process going forwards.